.gitmodules

```diff
@@ -16,3 +16,4 @@
 [submodule "lib/webauthn-sol"]
 	path = lib/webauthn-sol
 	url = https://github.com/base-org/webauthn-sol
+	branch = wilson/fix
```

Comment:
is there some change that still needs to be merged upstream?

Reply:
oops, need to clean up